synfig-studio/src/gui/CMakeLists.txt

```diff
@@ -18,6 +19,7 @@ add_definitions(-DIMAGE_DIR="${DATA_PREFIX}/share/pixmaps")
 add_definitions(-DSYNFIG_DATADIR="${SYNFIG_DATA_PREFIX}")
 add_definitions(-DLOCALEDIR="${DATA_PREFIX}/share/locale")
 add_definitions(-DCMAKE_BUILD)
+add_definitions(-DDEVEL_VERSION)
```

Comment:
`add_definitions(-DDEVEL_VERSION="...")` will make the code short and more understandable. There's no need to overcomplicate that process by using `synfig-studio/src/gui/autorevision.h.cmake.in`.

Reply:
Remove the line I added in `synfig-studio/src/gui/autorevision.h.cmake.in` and add `add_definitions(-DDEVEL_VERSION="Revision date: ${GIT_DATE}\nBranch: ${GIT_BRANCH}\nRevision ID: ${GIT_HASH}")` make the compilation failed.

I do not understand yet about this. But I think it because of the newline (`\n`) in the DEVEL_VERSION string. If I pass `add_definitions(-DDEVEL_VERSION="Revision date: ${GIT_DATE} Branch: ${GIT_BRANCH} Revision ID: ${GIT_HASH}")`, the compilation success, but it is not what we want. Any idea how to pass `\n` in add_definitions()? I have not found any good reference.